loleaflet/src/core/Socket.js

```diff
@@ -273,20 +273,36 @@ L.Socket = L.Class.extend({
 			this._map._docLayer.pauseDrawing();
 		}
 		// console.log2('Slurp events ' + that._slurpQueue.length);
-		for (var i = 0; i < this._slurpQueue.length; ++i) {
+		var queueLen = this._slurpQueue.length;
+		var complete = true;
+		for (var i = 0; i < queueLen; ++i) {
 			var evt = this._slurpQueue[i];
-			try {
-				// it is - are you ?
```

Comment:
ah you removed my joke about being on-message ;-)

Reply:
restored ;)